public/algorytm.php

```diff
@@ -1,6 +1,136 @@
-<?php 
-/*
-	Algorytm przetwarzania obrazu ze skali szarości na kolor z wykorzystaniem mapowania pikseli.
-*/
+<?php
+/**
+ * createImage()	- Tworzy nowy obraz z pliku
+ * @param string	$filename nazwa pliku
+ * 
+ * @return zwraca obraz lub w przypadu niepowodzenia false;
+ */
+function createImage($filename){
+		switch (exif_imagetype($filename)) {
+			case IMAGETYPE_GIF:
+				return imagecreatefromgif($filename);
+				break;
```

Comment:
samo `return` już wyjdzie z funkcji, więc nie potrzeba dalej `break`. Ale to tylko moje czepialstwo, nie musisz poprawiać :smiley: